js/html/Index.svelte

```diff
@@ -37,10 +38,11 @@
 	/>
 	<div class:pending={loading_status?.status === "pending"}>
 		<HTML
-			min_height={loading_status && loading_status?.status !== "complete"}
 			{value}
 			{elem_classes}
 			{visible}
+			{min_height}
```

Comment:
Just to understand, why do we need to pass `min_height` into the HTML Svelte component, when we are already applying it to the outer `Block` component in line 25 above? It feels redundant to me